scripts/Dockerfile

```diff
@@ -1,18 +1,31 @@
-FROM jupyter/minimal-notebook
+FROM jupyter/base-notebook:python-3.9.12
 
-ARG SRC_DIR=.
+LABEL name="pymc"
+LABEL description="Environment for PyMC version 4"
 
-MAINTAINER Austin Rochford <austin.rochford@gmail.com>
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 
-ADD $SRC_DIR /home/jovyan/
-RUN /bin/bash /home/jovyan/scripts/create_testenv.sh --global --no-setup
+# Switch to jovyan to avoid container runs as root
+USER $NB_UID
 
-#  matplotlib nonsense
-ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
-ENV MPLBACKEND=Agg
-# for prettier default plot styling
-RUN pip install seaborn
-# Import matplotlib the first time to build the font cache.
-RUN python -c "import matplotlib.pyplot"
+COPY /conda-envs/environment-dev-py39.yml .
+RUN mamba env create -f environment-dev-py39.yml && \
+    /bin/bash -c ". activate pymc-dev-py39 && \
+    mamba install -c conda-forge -y pymc" && \
+    conda clean --all -f -y
 
-ENV PYTHONPATH $PYTHONPATH:"$HOME"
+# Fix PkgResourcesDeprecationWarning
+RUN pip install --upgrade --user setuptools==58.3.0
+
+#Setup working forder
+RUN mkdir /home/jovyan/workspace
+WORKDIR /home/jovyan/workspace
```

Comment:
I can double check myself tomorrow, but I *think* the jupyter base image makes a directory `/home/jupyter/work` that you're supposed to use as a mount target.  If so, I think you could remove 20-22.  (also very small spelling nit: forder -> folder on line 20)

Reply:
Hi yes, I will check it as well :)

Reply:
Yup just build the container, everything works great and mounts correctly.  There is both `work` and `workspace` in there though.  It's mentioned very briefly in their docs here: https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#additional-runtime-configurations.   It'd be a little neater to remove `RUN mkdir /home/jovyan/workspace` and use `WORKDIR /home/jovyan/work` instead.

Also, just a heads up that `jovyan` *should* be replaced with `$NB_USER` in your Dockerfile, since that is [something that's configurable too](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#user-related-configurations).  I think it adds unnecessary complication to `scripts/docker_container.sh` though for this case, but might come in handy in other contexts.  Just want to mention if by chance you hadn't seen it!